queue_services/account-mailer/src/account_mailer/enums.py

```diff
@@ -28,6 +28,7 @@ class SubjectType(Enum):
     PAD_SETUP_FAILED = '[BC Registries and Online Services] Your Account is Temporarily Suspended'
     PAYMENT_PENDING = '[BC Registries and Online Services] Payment is now due for pending transaction on your account'
     EJV_FAILED = 'GL disbursement failure for EJV'
```

Comment:
this needs to be fixed, it's not just a disbursement error? 
EJV payments can have issues too?